### PR TITLE
Redirect /, do it in config, and rename rasterize.js -> index.js

### PR DESCRIPTION
**Pull request by paulmelnikow**
@styfle pointed out that my tweet linked to https://raster.shields.io/ which is a 404. This redirects it to https://shields.io/. It also moves the redirect into config.

This makes it less portable (i.e. it won't work if you're self-hosting on your own infrastructure), though that may not be a big deal

### CODE REVIEW

File: now.json [hunk 1]
```diff
@@ -12,10 +12,18 @@
...
     {
-      "src": "rasterize.js",
+      "src": "index.js",
       "use": "@now/node",
       "config": { "maxLambdaSize": "40mb" }
     }
   ],
-  "routes": [{ "src": "/(.*)", "dest": "/rasterize.js" }]
+  "routes": [
+    { "src": "/(.*)", "dest": "/index.js" },
```
Comment (styfle) [inline now.json]:
```suggestion
    { "src": "/(.+)", "dest": "/index.js" },
```

The order here matters so you should move this below the root redirect `/`. Also the regex can be changed to help

**Comment by paulmelnikow** [comment]
Confirmed that https://svg-to-image-proxy-ef60rlco3.now.sh/ redirects to https://shields.io/ and https://svg-to-image-proxy-ef60rlco3.now.sh/favicon.ico gives a 404 page (i.e. not the error badge).

**Comment by paulmelnikow** [comment]
Hey @calebcartwright, want to try to merge + deploy this change?

**Review by calebcartwright — APPROVED** [review]

**Comment by calebcartwright** [comment]
Sure

**Comment by calebcartwright** [comment]
alright i just ran it, might wanna check to make sure I didn't blow it up 😄

**Comment by paulmelnikow** [comment]
It worked! And so is the redirect: https://raster.shields.io/

Except weirdly it's saying I did it:

Screen Shot 2019-07-07 at 3 33 19 PM

**Comment by paulmelnikow** [comment]
And OMG I love not being the bottleneck!

**Comment by styfle** [comment]
Did you set up a team on ZEIT for this deployment? If not, then it might only be connected to your personal account instead of the whole org.

**Comment by paulmelnikow** [comment]
Yea, this is in the Shields1 team.

The aliasing to `raster.shields.io` doesn't happen in GitHub – AFAIK @calebcartwright aliased it from the CLI. (Is that right, Caleb?)

**Comment by calebcartwright** [comment]
> (Is that right, Caleb?)

Correct. Following the deploy command we have in the [docs](https://github.com/badges/svg-to-image-proxy#rastershieldsio-production-hosting) (`now alias ...`)

**Comment by calebcartwright** [comment]
FWIW, on our dashboard I see the Alias that I did (https://zeit.co/shields1/svg-to-image-proxy/aliases) but the deployment shows under Paul's id (which I'd expect given the author of this PR)

**Comment by paulmelnikow** [comment]
Huh. Right, so this is what I see under Aliases too:

Screen Shot 2019-07-07 at 9 09 22 PM

The first one of those aliases, `raster.shields.io` was created by Caleb from the CLI. The other four of those aliase were created by Now for GitHub based on my last commit in this PR.

However under Events, it's showing the `raster.shields.io` alias was created by me on GitHub, when actually it was Caleb. Further down, it's showing the GitHub-triggered aliases, correctly.

Screen Shot 2019-07-07 at 9 12 46 PM

**Comment by styfle** [comment]
@paulmelnikow This might be a bug when retreiving a merge commit with both an author and committer (as seen in this merged commit 75a3688c648443b640be368a7f5a5b33b8c21627).

image

I notified the ZEIT team, specifically @arunoda who is the expert with GitHub integrations, thanks!

**Comment by paulmelnikow** [comment]
It's a little confusing, because that alias was made manually from the CLI, _not by the GitHub integration_. I wouldn't expect the users attached to the commit, whether there are one or two of them, to matter at all.

Thank you for passing along the report!

**Pull request merged**